commit 465f896d1524cbde1ddd8534e3a7d04d50bb08fc
Author: Alexander K <ak@natsys-lab.com>
Date:   Sun Jan 6 20:37:53 2019 +0300

    Remove unnecessary comment after fix of TCP socket write memory accounting
    on Tempesta TLS side.

diff --git a/net/ipv4/tcp_output.c b/net/ipv4/tcp_output.c
index 4e79cb5e..b676c6a5 100644
--- a/net/ipv4/tcp_output.c
+++ b/net/ipv4/tcp_output.c
@@ -2374,10 +2374,6 @@ static bool tcp_write_xmit(struct sock *sk, unsigned int mss_now, int nonagle,
 				net_warn_ratelimited(
 					"Tempesta: cannot encrypt data (%d),"
 					" reset a TLS connection.\n", result);
-				/*
-				 * FIXME #984 WARNING: at net/core/stream.c:205
-				 * sk_stream_kill_queues+0x106/0x120
-				 */
 				tcp_reset(sk);
 				break;
 			}
